index.html

```diff
@@ -22,7 +22,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/public/icons/favicon-16x16.png">
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
+    <title>Raia</title>
```

Comment:
Consider updating the title to reflect the content accurately.

Suggested change:
```html
<title>Raia</title>
``` 

For reference, see [HTML Title Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title).